hnn_core/network_builder.py

```diff
@@ -203,6 +206,7 @@ def _create_parallel_context(n_cores=None):
 
         # be explicit about using fixed step integration
         _CVODE.active(0)
+        _CVODE.use_fast_imem(1)
```

Comment:
I dropped this here a bit haphazardly when I first set up the electrode API. I am concerned that it may slow down simulations that don't have LFP recordings, as I believe this will start a current recording for all segments: https://www.neuron.yale.edu/neuron/static/py_doc/simctrl/cvode.html#CVode.use_fast_imem

If it the difference is negligible then it's fine, but it may be worth passing a boolean indicating if LFP electrodes exist.

Reply:
Great point!